src/service/apiClient.js

```diff
@@ -69,6 +69,11 @@ async function editPost(postId, updatedContent) {
  
 }
 
+async function toggleLike(postId) {
+  const response = await post(`posts/${postId}/like`, {}, true); // Empty object `{}` as no body is required for this action
```

Comment:
`null` is better for the data field if we don't need it. Take a look at how the `request` function works in the `apiClient` to see why!